### ISSUE TITLE
Finetune Model

### ISSUE THREAD

**Issue by Dania-github** [abdur75648/UTRNet-High-Resolution-Urdu-Text-Recognition#11]
how to finetune a model using FT argument ?

**Comment by abdur75648** [comment]
Hi @Dania-github, if using this argument, the model will load the provided checkpoint and resume training from there (Ref. [train.py#L94).](https://github.com/abdur75648/UTRNet-High-Resolution-Urdu-Text-Recognition/blob/049c2d664617e6a5271ffb444c53a7d8599741cf/train.py#L94) Make sure to adjust the learning rate accordingly.
